scripts/packing_similarity_dendrogram/packing_similarity_dendogram.py

```diff
@@ -366,6 +373,7 @@ def main(input_file, matrix_file, n_ps_mols, output_ps_results, conf_threshold,
     # Generate a cluster hierarchy - populate it initially with every structure
     cluster_list = []
     for i in range(0, structure_size):
+#        cluster_list.append({'level': n_ps_mols, 'identifiers': [i+1], 'children': []})
```

Comment:
You can just delete the code if you are commenting it out, the version control will remember it :D